.vscode/launch.json

```diff
@@ -0,0 +1,16 @@
+{

```

Comment:
why do you want to add these .vscode files to the repo?

Reply:
It is useful for those who use VSCode to have preconfigured env :)